sjsonnet/src-jvm/sjsonnet/SjsonnetMain.scala

```diff
@@ -99,34 +134,14 @@ object SjsonnetMain {
       config.varBinding,
       config.tlaBinding,
       OsPath(wd),
-      importer = resolveImport(
-        config.jpaths.map(os.Path(_, wd)).map(OsPath(_)),
-        allowedInputs
-      ),
+      importer = importer match{
+        case Some(i) => (wd: Path, str: String) => i(wd, str).map(p => (OsPath(p), os.read(p)))
+        case None => resolveImport(config.jpaths.map(os.Path(_, wd)).map(OsPath(_)), allowedInputs)
+      },
```

Comment:
This fixes a regression where we accidentally stopped using `importer` once we started using `config.jpaths` to validate imports. This change makes it such that `importer` takes precedence over `config.jpaths` if passed in